lib/src/footer/Footer.stories.tsx

```diff
@@ -121,6 +126,16 @@ export const Chromatic = () => (
         </div>
       </DxcFooter>
     </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Reduced" theme="light" level={4} />
+      <DxcFooter mode="reduced">
+        {info.map((tag, index) => (
+          <div key={`tag${index}${tag.label}${tag.text}`}>
```

Comment:
Layout components should be used for showing children, since this is controlled by the user.